README.md

```diff
@@ -4,26 +4,26 @@
 
 This repository contains the [System.Device.Gpio](https://www.nuget.org/packages/System.Device.Gpio) library and implementations for various boards like [Raspberry Pi](https://www.raspberrypi.org/) and [Hummingboard](https://www.solid-run.com/nxp-family/hummingboard/).
 
-The repository also contains [IoT.Device.Bindings](https://www.nuget.org/packages/Iot.Device.Bindings), a growing set of community-maintained [device bindings](src/devices/README.md) for IoT components.
+The repository also contains [Iot.Device.Bindings](https://www.nuget.org/packages/Iot.Device.Bindings), a growing set of community-maintained [device bindings](src/devices/README.md) for IoT components.
```

Comment:
this is fine in this PR but in the future try to avoid changing stuff unrelated to the issue you're fixing for couple of reasons:
- it's much easier to port/revert changes to stable branches
- it makes it easier to review since changeset is isolated
- you're more likely to get better review since it's easier to see the problem you're solving